docs_guidelines/apidocs/api.examples.rst

```diff
@@ -0,0 +1,21 @@
+######################
+API Reference Examples
+######################
+
+Module API
+==========
+
+* `qiskit.circuit <https://qiskit.org/documentation/apidoc/circuit.html>`_
+* `qiskit.visualization <https://qiskit.org/documentation/apidoc/visualization.html>`_
+* `qiskit.providers.fake_provider <https://qiskit.org/documentation/apidoc/providers_fake_provider.html>`_
+
+Code Examples
```

Comment:
what do you mean by code examples? How are these examples different from the ones above? what differences should I (as the reader) be paying attention to? I think a few words about this would be helpful